modules/swagger-codegen/src/main/java/io/swagger/codegen/v3/examples/ExampleGenerator.java

```diff
@@ -261,8 +268,8 @@ private Object resolveModelToExample(String name, String mediaType, Schema schem
         } else if (schema.getProperties() != null) {
             logger.debug("Creating example from model values");
             for (Object propertyName : schema.getProperties().keySet()) {
-                schema.getProperties().get(propertyName.toString());
-                values.put(propertyName.toString(), resolvePropertyToExample(propertyName.toString(), mediaType, schema, processedModels));
+                Schema property = (Schema) schema.getProperties().get(propertyName.toString());
+                values.put(propertyName.toString(), resolvePropertyToExample(propertyName.toString(), mediaType, property, processedModels));
```

Comment:
This was the main bug fix that caused things to start working again, i.e. property values being populated instead of `""`.